### PR TITLE
Expose Delayed::Job metrics

### PR DESCRIPTION
**Pull request by emileswarts**
This is currently used to see how many submissions have been submitted to
Optics.

The format of this output allows us to easily feed this into Grafana in the
future.

### CODE REVIEW

File: app/controllers/metrics_controller.rb [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    }
+  end
+
+  def failed_stats
+    {
+      name: :delayed_jobs_failed,
+      docstring: 'Number of jobs failed',
+      value: Delayed::Job.where('attempts > 0').count
+    }
+  end
```
Comment (cob16) [inline app/controllers/metrics_controller.rb]: The business logic should be pulled out of the controller. Will make the tests easier for metrics

File: app/controllers/metrics_controller.rb [hunk 2]
```diff
@@ -0,0 +1,42 @@
+class MetricsController < ActionController::Base
```
Comment (cob16) [inline app/controllers/metrics_controller.rb]: Is it safe for this route to have no auth?

File: .rubocop.yml [hunk 3]
```diff
@@ -6,6 +6,7 @@ AllCops:
   - bin/**
   - db/schema.rb
   - Guardfile
+  - config/environments/**
```
Comment (emileswarts) [inline .rubocop.yml]: Was getting linted on LOC in the development config file which makes no sense

File: app/controllers/metrics_controller.rb (hunk 2 above)
Comment (emileswarts) [reply app/controllers/metrics_controller.rb]: Added basic auth

File: config/environments/development.rb [hunk 4]
```diff
@@ -1,15 +1,6 @@
...
-
-  # In the development environment your application's code is reloaded on
-  # every request. This slows down response time but is perfect for development
-  # since you don't have to restart the web server when you make code changes.
   config.cache_classes = false
-
-  # Do not eager load code on boot.
   config.eager_load = false
-
-  # Show full error reports.
```
Comment (emileswarts) [inline config/environments/development.rb]: This was an attempt to bypass a rubocop lint.
Forgot to revert, but ultimately don't think we need these comments, so leaving it out.

**Pull request merged**